apps/gauzy/src/app/pages/goals/goal-template-select/goal-template-select.component.ts

```
@@ -150,59 +149,71 @@
 				this.form.value.level === GoalLevelEnum.EMPLOYEE
-					? 'ownerEmployee'
+					? 'ownerEmployeeId'
 					: this.form.value.level === GoalLevelEnum.TEAM
-					? 'ownerTeam'
-					: 'organization'
-			] = this.form.value.owner;
+					? 'ownerTeamId'
+					: 'organizationId'
+			] = this.form.value.ownerId;
+
 			delete goal.owner;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)